packages/headless/src/features/question-answering/question-answering-analytics-actions.ts

```diff
@@ -301,53 +322,76 @@ export const logOpenSmartSnippetSuggestionSource = (
         }
       );
     },
-    analyticsType: 'itemClick',
-    analyticsPayloadBuilder: (state): ItemClick => {
+    analyticsType: 'Qna.CitationClick',
+    analyticsPayloadBuilder: (state): Qna.CitationClick => {
       const relatedQuestion = relatedQuestionSelector(
         state,
         payload.questionAnswerId
       );
-      const result = answerSourceSelector(state, relatedQuestion?.documentId)!;
-      const information = partialDocumentInformation(result, state);
       return {
-        searchUid: state.search?.response.searchUid ?? '',
-        position: information.documentPosition,
-        itemMetadata: analyticsEventItemMetadata(result, state),
+        answer: {
+          responseId: state.search?.response.searchUid || '',
+          type: 'SmartSnippetSuggestion',
+        },
+        citation: {
+          // Question what should be the citation id for SS.
+          id: relatedQuestion?.documentId.contentIdValue || '',
```

Comment:
@jstpierref, In the case of smart snippets, what's the value expected to be received as the key `id` in the `citation` object, 
is it the permanentId of the result from where the snippet was extracted?

Reply:
Yes, that would be the `permanentId`, even if it can be shared among many related questions